app/controllers/review_controller.rb

```diff
@@ -79,9 +79,9 @@ def submit_for_2i
         review_requester_id: current_user.uid,
         status: status,
       )
-        generate_change_note(status, params[:additional_comments])
+        generate_change_note(I18n.t!("step_by_step_page.statuses.#{status}"), params[:additional_comments])
 
-        redirect_to step_by_step_page_path(@step_by_step_page.id), notice: "Step by step page was successfully #{status.humanize.downcase}."
+        redirect_to step_by_step_page_path(@step_by_step_page.id), notice: "Step by step page was successfully submitted for 2i review."
```

Comment:
Probably out of scope for this PR, but these success notices feel like they should be held as translations too.  Maybe something we can do if there's time at the end of the mission.

Reply:
Yup would be a good step - it's always rather painful when you're transitioning and there's not clear logic of what is in or isn't.

>  if there's time at the end of the mission
😆